tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php

```diff
@@ -95,7 +95,7 @@ public function testConfigureIncludeSets(
         $this->fixer->configure(['include' => $include]);
 
         if (null === $expectedExceptionClass) {
-            $this->addToAssertionCount(1);
+            $this->expectNotToPerformAssertions();
```

Comment:
this one was correct, if there is no exception expected and it was not thrown than it did pass an assertion so increasing the count seems valid to me